.github/workflows/snyk-monitor.yml

```diff
@@ -3,15 +3,20 @@ name: Snyk monitor workflow for tracking third-party dependencies
 on:
   push:
     branches: [ "master" ]
+
 jobs:
   snyk-monitor:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@master
+      - name: Install pdm
+        run: pip install --user pdm
+      - name: Convert pdm.lock to requirements.txt
+        run: pdm export -o requirements.txt 
       - name: Run Snyk to check for vulnerabilities
         uses: snyk/actions/python@master
         env:
           SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
         with:
           command: monitor
-          args: --project-name=jf_agent --org=jellyfish-k9n
\ No newline at end of file
+          args: --project-name=jf_agent --org=jellyfish-k9n --file=requirements.txt
```

Comment:
Are the generated `requirements.txt` rows just formatted as `package==1.2.3`?

Reply:
Almost, except it includes the hashes too + a CLI arg. Here's an shortened example:

```
# This file is @generated by PDM.
# Please do not edit it manually.

appdirs==1.4.4 \
    --hash=sha256:7d5d0167b2b1ba821647616af46a749d1c653740dd0d2415100fe26e27afdf41 \
    --hash=sha256:a841dacd6b99318a741b166adb07e19ee71a274450e68237b4650ca1055ab128
attrs==23.1.0 \
    --hash=sha256:1f28b4522cdc2fb4256ac1a020c78acf9cba2c6b461ccd2c126f3aa8e8335d04 \
    --hash=sha256:6279836d581513a26f1bf235f9acd333bc9115683f14f7e8fae46c98fc50e015
...

--index-url https://pypi.python.org/simple
```

Reply:
Sweet, the inclusion of hashes were exactly what I was wondering about 🥳